tests/std/tests/GH_002769_handle_deque_block_pointers/test.cpp

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cstddef>
+#include <deque>
+#include <iterator>
+#include <memory>
+#include <type_traits>
+
+using namespace std;
+
+size_t fancy_counter{};

```

Comment:
I would prefer to use 0 here 
```suggestion
size_t fancy_counter = 0;
```